CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## [6.0.0](https://github.com/plivo/plivo-java/tree/v6.0.0) (2024-05-02)
```

Comment:
Please confirm if the new version will be "5.41.0" or "6.0.0". As it a feature release, it should be "5.41.0"